green-commute/src/main/resources/application.properties

```diff
@@ -1,4 +1,6 @@
 server.port=8000
 spring.datasource.url=jdbc:mysql://localhost:3306/green_commute_db?useSSL=false&serverTimezone=UTC
 spring.datasource.username=ganesh
-spring.datasource.password=Ganesh@123
\ No newline at end of file
+spring.datasource.password=Ganesh@123
```

Comment:
try storing passwords in env file.
In real world we should never commit secrets like passwords to git.

Reply:
db info  added to environment variables